frontend/src/pages/NewRunV2.tsx

```diff
@@ -78,6 +78,8 @@ interface RunV2Props {
   namespace?: string;
   existingRunId: string | null;
   apiRun: ApiRunDetail | undefined;
+  originalRecurringRunId: string | null;
```

Comment:
Question: Can we use `originalRecurringRunId?: string` instead?

Reply:
If we change to undefined, we have to add an inline logic `originalRecurringRunId={originalRecurringRunId ? originalRecurringRunId : undefined}  in `NewRunSwitcher`, because the type of returned object from urlParser.get() is `string | null`. I tend to keep the current interface definition.